conf.d/done.fish

```diff
@@ -111,7 +111,7 @@ function __done_is_tmux_window_active
     # ppid == "tmux" -> break
     set tmux_fish_pid $fish_pid
     while set tmux_fish_ppid (ps -o ppid= -p $tmux_fish_pid | string trim)
-            and test ! (basename (ps -o exe= -p $tmux_fish_ppid)) = "tmux"
+            and string match -r -q "tmux*" (basename (ps -o exe= -p $tmux_fish_ppid))
```

Comment:
Looks like you are using glob while mistakenly specify regex (`-r`) flag. Either remove `-r` or change the pattern to `tmux.*`.